server.js

```diff
@@ -14,10 +21,81 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
+app.get("/", function (request, response)
+{
   response.sendFile(__dirname + "/index.html");
 });
 
 app.listen(process.env.PORT);
+
+
+
+
+
+app.get("/messages", (req, res) =>
+{
+  res.send(messages);
+});
+
+app.post("/messages", (req, res) =>
+{
+  let msgName = req.body.from;
+  let msgText = req.body.text;
+  let idPosition = messages.length;
```

Comment:
These values don't change, so you can declare them as `const` instead. 

The answers on this SO question give a good overview of why this is a good thing to do : https://stackoverflow.com/questions/136880/sell-me-on-const-correctness